flexx/app/_funcs.py

```diff
@@ -189,7 +189,12 @@ def init_notebook():
     
     # We set session_id and app_name in a way that it does not end up
     # in exported notebook.
-    url = 'ws://%s:%i/flexx/ws/%s' % (host, port, session.app_name)
+    if config.ssl_certfile != '' or config.ssl_keyfile != '':
+        # Connect over SSL
+        url = 'wss://%s:%i/flexx/ws/%s' % (host, port, session.app_name)
+    else:
+        url = 'ws://%s:%i/flexx/ws/%s' % (host, port, session.app_name)
```

Comment:
@alexveden ah, you're right. In that case: `proto = 'wss' if server.proto == 'https' else 'ws'`